index.js

```diff
@@ -0,0 +1,59 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let select = document.querySelector("select")
+    let form = document.querySelector("#form")
+    let ul = document.querySelector("#ul")
+    let submit = document.querySelector("#submit")
+    let userInput = document.querySelector("#userInput")
+    let movieDiv1 = document.querySelector("#movieDiv1")
+    let movieDiv2 = document.querySelector("#movieDiv2")
+    let movieDiv3 = document.querySelector("#movieDiv3")
+    let movieData = []
+    
+    const filmFetch = async () => {
+        try {
+            let res = await axios.get(`https://ghibliapi.herokuapp.com/films/`)
+            let movies = res.data
+            
+            movies.forEach(film => {
```

Comment:
You want to execute this when a change is happening using an event listener on the select